src/lib/common/schema.ts

```diff
@@ -119,13 +119,24 @@ export const contentEntryField = sqliteTable(
     valueString: text("value_string"),
     valueBool: integer("value_bool", { mode: "boolean" }),
     valueKeywords: text("value_keywords"),
-    valueDate: integer("value_date", { mode: "timestamp_ms" }),
+    valueDate: text("value_date"),
     valueNumber: text("value_number"),
     valueObjects: text("value_objects"),
   },
   (table) => {
     return {
       pk: primaryKey({ columns: [table.id, table.draftContent] }),
+      datax: unique().on(
```

Comment:
i will removed the unique index here since it think this should be handled not by a unique index rather by the insert query based on the field type